sportnet/sportnet/playgrounds/tests/test_models.py

```diff
@@ -59,3 +35,38 @@ def test_delete(self):
         self.assertEqual(len(models.Sport.objects.all()), 2)
         models.Sport.objects.all().first().delete()
         self.assertEqual(len(models.Sport.objects.all()), 1)
+
+
+class PlaceEventTest(TestCase):
+
+    def setUp(self):
+        call_command('flush', '--noinput')
+        self.first_test = PlaceEvent(city='Moscow', street='Baumanskaya')
+        self.first_test.save()
+        self.assertEqual(len(PlaceEvent.objects.all()), 1)
+
+    def test_create(self):
+        self.assertEqual(self.first_test.city, 'Moscow')
+        self.assertEqual(self.first_test.street, 'Baumanskaya')
+        self.assertEqual(len(PlaceEvent.objects.all()), 1)
+
+    def test_read(self):
+        city_test = PlaceEvent.objects.all().first().city
+        street_test = PlaceEvent.objects.all().first().street
+        self.assertEqual(city_test, 'Moscow')
+        self.assertEqual(street_test, 'Baumanskaya')
+
+    def test_update(self):
+        city_test = PlaceEvent.objects.all().first()
```

Comment:
```suggestion
        place = PlaceEvent.objects.all().first()
```
Ниже соответственно тоже переименовать